tasks/2_convert-to-f/convert-to-f.js

```diff
@@ -0,0 +1,4 @@
+export const convertToF = (celsius) => {
+  let fahrenheit = celsius*9/5 + 32;
```

Comment:
spaces `celsius * 9 / 5 + 32;`

Reply:
also round-error can easliy happen here: https://floating-point-gui.de/ - it's not neccesary for this task but keep it in mind for future ;)